kernel/examples/kernel-examples/src/main/java/io/delta/kernel/integration/ReadIntegrationTestSuite.java

```diff
@@ -199,6 +199,22 @@ public void runTests() throws Exception {
                     asList(new Column("as_int"), Literal.ofInt(1))
                 )),
             0 /* expected row count */);
+
+        // Type widening: table with various type changes.
```

Comment:
these release verification tests are great! Can you add these to the `DeltaTableReadsSuite.scala` which are run as part of the Kernel tests?

Reply:
Or verifying the result is a problem? If yes, may be select one or two rows using filter and hardcode the expected values.

Reply:
Good call out, I missed `DeltaTableReadsSuite`, I added tests there